Package.swift

```diff
@@ -15,7 +15,7 @@ let package = Package(
         ),
         .package(
             url: "https://github.com/apple/sourcekit-lsp",
-            revision: "12da8e5f54809b642701dd0dd6e145d3e0c67bc4"
+            branch: "main"
```

Comment:
We should always use specific revisions / tags, otherwise the project may silently break. The revision containing your fix is `5df8f3d9ac0e647238ed4203e8f399ae5a095aa3` so I think we should go for that one for now. Although I would prefer if this was on a separate PR just to make things easier in case we have to revert this one for some reason.

Reply:
I merged my other PR which was also modifying it, so it should be fine now if you rebase 👍